services/postgresql_manager.py

```diff
@@ -0,0 +1,83 @@
+import pkg_resources
+import requests
+import psycopg
+from psycopg import sql
+from configparser import ConfigParser
+
+# url as param -> received from Go?
+req = requests.get()
```

Comment:
This python program should listen to http, and respond to requests coming from HTTP get requests.